CHANGELOG.md

```diff
@@ -1,3 +1,64 @@
+Version 1.15.1
+==============
+**September 28, 2022**
+
+This release contains a small number of important bug fixes. Many thanks
+to all our new contributors @MarcSkovMadsen, @j-svensmark, @ceball,
+@droumis, @ddrinka and @stanwest as well as a special thanks to @Hoxbro
+for his many bug fixes. An additional thank you goes out to @philippjfr,
+@jbednar and @jlstevens.
+
+Enhancements:
+
+- Return samples from `decimate` operation in order
+  ([#5452](https://github.com/holoviz/holoviews/pull/5452))
```

Comment:
```suggestion
- Sort output of `decimate` operation so that it can be used with connected Elements
   (Curve, Area, etc.) ([#5452](https://github.com/holoviz/holoviews/pull/5452))
```